tests/test_dsl.py

```diff
@@ -79,80 +82,55 @@ class OldCohort(OldCohortWithPopulation):
 def test_count_aggregation(cohort_with_population):
     class OldCohort(OldCohortWithPopulation):
         # Define tables of interest, filtered to relevant values
-        num_events = table("clinical_events").count("code")
+        num_events = table("clinical_events").filter("code", not_equals=None).count()
```

Comment:
Why do you have to change the old version?

Reply:
Both `table("clinical_events").count("code")` and `table("clinical_events").filter("code", not_equals=None).count()` should return the same data, but they have different representations in the query model.

It might be possible to get `table("clinical_events").filter("code", not_equals=None).count()` to produce the simpler query model representation, but I'm going to go with [@rebkwok's suggestion to simplify this](https://github.com/opensafely-core/cohort-extractor-v2/pull/271/commits/a97955e73303d91536a08ced9640a69b26f9ae8f#r755904449).

Reply:
Addressed in 418e812e2bb1b71d28a7f9a2a628b3f5e4034d06.